.github/workflows/ci.yml

```diff
@@ -3,11 +3,17 @@ name: CI
 on:
   push:
   pull_request:
+  schedule:
+    # Run daily at 00:00 so we get notified if CI is broken before a pull request
+    # is submitted.
+    - cron:  '0 0 * * *'
```

Comment:
QUESTION: How does this interact with the conditional check `if: github.event_name == 'push' || github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id` ? Do we need another condition?

Reply:
nice catch! ⚡  I'm not sure anymore about this implementation with mentioned conditions
I'll test conditions separately and provide feedback asap

Reply:
Good catch! - I think we need another condition like
```
if: github.event_name == 'push' || github.event_name == 'schedule' || github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id
```

Reply:
I have adapted all the conditions to make sure we run all the jobs, with `ILGPU_CLEAN_TESTS=true`, and on macOS too.

Reply:
thanks for covering `schedule` conditions @jgiannuzzi ⚡